### PR TITLE
Add a cozy-mjml package for email templating

### PR DESCRIPTION
**Pull request by nono**
Cozy-mjml is a new package for sharing MJML components between mail templates in several projects. Currently, I've just added a single custom component to show how it will work later.

### CODE REVIEW

**Comment by nono** [comment]
[(𝐌) [NUA] ETU j'ai des emails personnalisé en fonction de mon offre](https://trello.com/c/vXXMFYl5/1071-%F0%9D%90%8C-nua-etu-jai-des-emails-personnalis%C3%A9-en-fonction-de-mon-offre)

File: packages/cozy-mjml/src/hacks/datauri-index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// This file replaces node_modules/datauri/index.js, because the shebang
+// confuses webpack.
+module.export = require('datauri/module')
```
Comment (ptbrowne) [inline packages/cozy-mjml/src/hacks/datauri-index.js]: webpack has a shebang loader  : https://www.npmjs.com/package/shebang-loader

File: packages/cozy-mjml/src/components/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const core = require('mjml-core')
+
+const components = [require('./MJCozyTitle')]
+
+components.forEach(component => {
+  core.registerComponent(component)
+})
```
Comment (ptbrowne) [inline packages/cozy-mjml/src/components/index.js]: I am not a fan of side-effects in index.js. I understand the usecase here but I think I would prefer to be explicit and export a `register` function.

File: packages/cozy-mjml/test/components.spec.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// Loads the MJML custom components, by side effect
+require('../src/components')
```
Comment (ptbrowne) [inline packages/cozy-mjml/test/components.spec.js]: Thus here, `const registerComponents = require('../src/components').register; registerComponents()` ?

File: packages/cozy-mjml/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+etc.).
+
+## How it works?
+
+Cozy-MJML exposes a single binary that behave like mjml CLI. The goal is to
+replace the mjml binary by this one on the cloudery: it will allow to use our
+custom MJML components with mjml-rails, even if the mjml CLI doesn't support
+custom components and mjml-rails has some expectations (like the version number
+of the CLI tools). This binary will also be used by the cozy-stack to generate
+HTML mails: the stack will run it like a konnector or a service.
```
Comment (ptbrowne) [inline packages/cozy-mjml/README.md]: Should the email service be an always running server ? I guess the time to generate an email would be then lower than when starting the node process from scratch every time.

Comment (nono) [reply packages/cozy-mjml/README.md]: There are some trade-offs: having an always running server would be faster, but it makes things more complicated to deploy (for self-hosted in particular), monitor, debug. And we don't send that many mails that the performance would be noticeable.

**Pull request merged**